src/index.ts

```diff
@@ -5,9 +5,11 @@ import ShopifySession from './auth/session';
 import ShopifyClients from './clients';
 import ShopifyUtils from './utils';
 import ShopifyWebhooks from './webhooks';
+import {LATEST_API_VERSION} from './base-types';
 
 export const Shopify = {
   Context,
+  LATEST_API_VERSION,
```

Comment:
I haven't tested this, but I suspect we may already be exporting this in line 22 below:

```ts
export * from './types';
```

If not, I think it might be best to export it from there.